lib/graphql/schema.flow.js

```diff
@@ -1,4 +1,4 @@
-/* @flow */
+// @flow 
```

Comment:
Why changing the comment syntax ? Both are fine, just wondering :)

Reply:
I just ran `yarn schema:generate`  🤔